server/src/schema.js

```diff
@@ -0,0 +1,68 @@
+const {gql} = require('apollo-server');
+
+const typeDefs = gql`
+    type Query {
+        events(pageSize: Int, after: String): EventConnection!
+        event(id: ID!): Event
+
+        # user information about the user currently logged in
+        me: User
+        user(id: ID!): User
+    }
+
+    type Mutation {
+        authGoogle(accessToken: AuthInput!): AuthResponse
+        authFacebook(accessToken: AuthInput!): AuthResponse
+        logout: Boolean! # success, then True.  
+    }
+
+    type AuthResponse {
+        token: String
+        name: String
+    }
+    input AuthInput {
+        accessToken: String!
+    }
+
+    type User {
+        id: ID!
+        firstName: String!
+        lastName: String!
+        email: String!
+        age: Int
+        sex: Sex
+        city: String
+        state: String
+        phone: String
+    }
+
+    type Event {
+        id: ID!
+        host: User!
+        title: String!
+        description: String!
+        price: Float!
+        # image: Upload!  not implemented in V0
+        maxParticipants: Int!
+        tags: [Tag]!
+    }
+
+    enum Sex {
```

Comment:
https://github.com/51bootcamp-2020/tinysocial/pull/2#discussion_r363180430
Maybe we can just remove gender information. Because I think our service is gender-agnostic?
 I'm gonna discuss this with our team more.